src/zeit/campus/article.py

```diff
@@ -0,0 +1,17 @@
+import zope.interface
+
+import zeit.cms.content.reference
+
+import zeit.campus.interfaces
+
+
+class TopicpageLink(zeit.cms.related.related.RelatedBase):
+
+    zope.interface.implements(zeit.campus.interfaces.ITopicpageLink)
+
+    topicpage = zeit.cms.content.reference.SingleResource(
+        '.head.topicpagelink', 'related')
+
+    label = zeit.cms.content.property.ObjectPathProperty(
```

Comment:
Eigentlich ist das doch "ich referenziere die Seite; als Label kommt deren Titel, aber an machen Verwendungsstellen will ich das überschreiben können". Von der Modellierung her wärs glaub ich passender, das mit nem Referenzobjekt zu machen (wie bei Bildreferenzen am Artikel, wo man die Caption und sowas auch lokal überschreiben kann), genau dafür sind die eigentlich da. Dann kann auch das Referenzobjekt schon den Fallback machen und man muss das nicht mehr in zeit.web auseinanderklamüsern.